checkstyle/import-control.xml

```diff
@@ -48,6 +48,7 @@
   <allow pkg="org.apache.kafka.common.utils" />
   <allow pkg="org.apache.kafka.common.errors" exact-match="true" />
   <allow pkg="org.apache.kafka.common.memory" />
+  <allow pkg="org.apache.kafka.coordinator.group.common"/>
```

Comment:
I suppose that we can remove this, isn't it?

Reply:
yep correct we don't need it now that we don't have any classes in it